src/minimal_unified_api.py

```
@@ -0,0 +1,232 @@
+            result = whisper_model.transcribe(tmp_file_path)
+            transcription = result["text"]
+
+            return {
+                "transcription": transcription,
+                "language": result.get("language", "unknown")
+            }
+        finally:
+            # Clean up temporary file
+            os.unlink(tmp_file_path)
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)